package.json

```diff
@@ -3,26 +3,25 @@
   "license": "SEE LICENSE IN LICENSE",
   "version": "0.1.1-alpha.1",
   "description": "React components for Osso",
-  "main": "umd/osso.js",
-  "module": "dist/index.esm.js",
+  "main": "dist/enterprise-oss-osso.cjs.js",
+  "module": "dist/enterprise-oss-osso.esm.js",
   "types": "dist/index.d.ts",
-  "author": "sbauch",
+  "sideEffects": false,
+  "author": "enterprise-oss",
   "scripts": {
-    "build": "rollup -c",
-    "clean": "rm -rf dist && rm -rf umd",
+    "build": "rollpkg build",
+    "clean": "rm -rf dist",
     "lint": "eslint ./src",
     "format": "tsc && eslint ./src --fix",
-    "start": "rollup -c --watch",
+    "prepare": "npm run build",
```

Comment:
this should enable us to install from github and avoid releasing so many pre-release versions to npm, but has the ugly side effect of adding build to every step of our CI process